idaes/core/surrogate/keras_surrogate.py

```diff
@@ -263,7 +263,9 @@ def save_to_folder(self, keras_folder_name):
               The name of the folder to contain the Keras model and additional
               IDAES metadata
         """
-        self._keras_model.save(keras_folder_name)
+        self._keras_model.save(
+            os.path.join(keras_folder_name, keras_model_name + ".keras")
+        )
```

Comment:
Suggestion: since Keras seems to support `pathlib.Path` objects now, this could be rewritten to be slightly more concise/readable (requires replacing `import os.path` with `from pathlib import Path` at the top of the file):

```suggestion
        self._keras_model.save(
            Path(keras_folder_name, keras_model_name).with_suffix(".keras")
        )
```